.kitchen.yml

```diff
@@ -0,0 +1,70 @@
+---
+driver:
+  name: docker
+  #socket: tcp://127.0.0.1:2376
+  hostname: git.ci.local
+  use_sudo: true
+
+provisioner:
+  name: salt_solo
+  salt_install: bootstrap
+  salt_bootstrap_url: https://bootstrap.saltstack.com
+  salt_version: latest
+  require_chef: false
+  #init_environment: |
+  #  # for reclass
+  #  mkdir -p $SALT_ROOT/reclass/classes
+  #  ln -fs /usr/share/salt-formulas/reclass/* $SALT_ROOT/reclass/
+  #  find /usr/share/salt-formulas/env/_formulas -name metadata -type d | xargs -I'{}' \
+  #    ln -fs {}/service $SALT_ROOT/reclass/classes/
+  vendor_repo:
```

Comment:
I would remove setup of APT repository as it's not going to be used (at least before we have custom PPA).
Also remove useless comments.